events/wmt23.md

```diff
@@ -198,43 +233,34 @@ In 2022, the *News* machine translation task was renamed the *General* machine t
 | 17:15 - 17:30 | [**The Devil Is in the Errors: Leveraging Large Language Models for Fine-grained Machine Translation Evaluation**](https://aclanthology.org/2023.wmt-1.100.pdf) <br>Patrick Fernandes, Daniel Deutsch, Mara Finkelstein, Parker Riley, André Martins, Graham Neubig, Ankush Garg, Jonathan Clark, Markus Freitag, Orhan Firat |
 
 
-## Scientific papers
+## Results
 
-### Topics
+### General task
 
-- Machine translation models (neural, statistical etc. )
-- Analysis of neural models
-- Using comparable corpora
-- Selection and preparation of data
-- Semi-supervised and unsupervised learning for machine translation, transfer learning
-- Multilingual machine translation
-- Incorporating linguistic information into machine translation
-- Machine translation inference
-- Manual and automatic methods for evaluating machine translation
-- Quality estimation
+Full results of the shared task: [*Findings of the 2023 Conference on Machine Translation (WMT23): LLMs Are Here but Not Quite There Yet*](https://aclanthology.org/2023.wmt-1.1.pdf)
 
-### Research Papers
-
-> Research papers should describe original research corresponding to the categories listed above. Research papers that have been or will be submitted to other meetings or publications must indicate this at submission time, and must be withdrawn from the other venues if accepted and published at WMT 2022.
-
-> We will not accept for publication papers that overlap significantly in content or results with papers that have been or will be published elsewhere. It is acceptable to submit work that has been made available as a technical report (or similar, e.g. in arXiv) without citing it.
-
-> For the research track, papers should be anonymised, be between 6 and 10 pages in length (excluding references) and may include supplementary material.
+The winner systems were listed according to their [average score](/human-evaluation-metrics#average-score-and-average-z-score).
+The results were determined with a bilingual direct assessment with [scalar quality metric](/human-evaluation-metrics#sqm) (SQM) with document context.
 
+#### → English
 
-### System Papers
+| Language pair | System | Average score |
+| --- | --- | --- |
+| German → | `GPT4-5shot` | 90.3 |
+| Chinese → | `Lan-BridgeMT`| 82.9 |
+| Japanese → | `GPT4-5shot` | 81.3 |
 
-> System papers must describe one or more shared task submissions. System paper submissions that we cannot link to a shared task submission will be rejected without review. System papers can overlap with other published work, and do not have to follow the double submission policy. There is no maximum length for system papers, but normally a short paper (4-6 pages) is appropriate. System papers should not be anonymised.
+#### English →
 
+| Language pair | System | Average score |
+| --- | --- | --- |
+| → German | `GPT4-5shot` | 89.0 |
+| → Czech | `ONLINE-W` | 84.1 |
+| → Chinese | `Yishu`| 82.2 |
+| → Japanese | `GPT4-5shot` | 79.5  |
 
-## Poster format
```

Comment:
I recommend removing this entire subsection, @bittlingmayer and @cefoo, as it contains very specific information that may not be relevant to the readers.